Automattic/jetpack
# Contact Form: ensure IP address is displayed for each submission

Fixes #29124

## Proposed changes:

Ensure we pull the feedback's IP address to display it in the legacy Feedback screen.

<img width="1129" alt="Screenshot 2024-07-16 at 17 56 58" src="https://github.com/user-attachments/assets/ef8ad410-08cd-453f-82ef-9570d866f92a">

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* On a new page, insert a Form block.
* View the page.
* Submit a form.
* Go to the Feedback in wp-admin.
* In the top right corner, ensure you are looking at the old legacy form list table, and not the newer inbox view.
* You should see your recent submission, with the IP address displayed (see screenshot above).
